### ISSUE TITLE
batch_normalization 층에서 "training" 파라미터의 의미

### ISSUE THREAD

**Issue by tygu1004** [tygu1004/promedius-summer-intern#2]
# 개요

배치 정규화 층 구현과정에서 training 파라미터 관련 문제가 있습니다.

# 세부사항

버전 1에서 tf.layers.batch_normalization 의 training 파라미터와
버전 2에서 tf.keras.layers.BatchNormalization 의 입력값으로 들어가는 training의 의미가 같은지 확인해야 합니다.
